### PR TITLE
changed readme

### PR DESCRIPTION
**Pull request by kdabug**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+[dallin]("./src/images/dammad-logo.png")[mikayda] \*\*
+\*\*[dallin: './src/images/dallin.jpeg'][mikayda: './src/images/mikaydapic.jpeg']
+
+# DAMMAD.IO
+
+We made mad apps in damn good time.
```
Comment (dallin-r-parker) [inline README.md]: I like this - should change **damn** to **dam**. Also on LinkedIn we say "Making DAM good software with MAD design" I liked your version of this, but we should stay consistent, so let me know what you think.

**Review by dallin-r-parker — COMMENTED** [review]
I will approve once you let me know what you want to do about the "slogan"

File: README.md (hunk 1 above)
Comment (kdabug) [reply README.md]: OO yes I forgot where we wrote that and was shooting from the hip - let me change it to the LinkedIn

**Review by dallin-r-parker — APPROVED** [review]
💯 🍖 - Nice!

**Pull request merged**